people/daniel_larimer.md

```diff
@@ -5,9 +5,9 @@ Location: US
 Name: [BitShares](../projects/bitshares.md)  
 Position: Founder  
 Name: [EOS](../projects/eos.md)  
-Position: Founder
+Position: Founder  
```

Comment:
Why null Dan Larimer changes here?

This PR is about Gnosis, not EOS or Steem.